src/coreclr/vm/method.hpp

```diff
@@ -2707,6 +2712,12 @@ class DynamicMethodDesc : public StoredSigMethodDesc
     PTR_DynamicResolver m_pResolver;
 
 public:
+
+#ifdef FEATURE_INTERPRETER
```

Comment:
```suggestion
#if defined(FEATURE_INTERPRETER) && !defined(FEATURE_PORTABLE_ENTRYPOINTS)
```
InterpreterPrecode is not a think with portable entrypoints.